includes/core.php

```diff
@@ -40,15 +39,6 @@ function setup() {
 	do_action( 'jobber_plugin_loaded' );
 }
 
-/**
- * Registers the default textdomain.
- */
-function i18n() {
-	$locale = apply_filters( 'plugin_locale', get_locale(), 'jobber-wp' );
-	load_textdomain( 'jobber-wp', WP_LANG_DIR . '/jobber-wp/jobber-wp-' . $locale . '.mo' );
-	load_plugin_textdomain( 'jobber-wp', false, plugin_basename( JOBBER_PLUGIN_PATH ) . '/languages/' );
```

Comment:
Loading the textdomain isn't required for plugins on WordPress.org, see https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#plugins-on-wordpress-org